README.md

```diff
@@ -9,7 +9,7 @@ Follow these steps to quickly get started with the [Plaid API](https://plaid.com
 1. [Sign up](https://dashboard.plaid.com/signup) with Plaid to get a set of API keys that are required for interacting with the API.
 2. Click the "Run in Postman" button below.
   
-[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/300b6d41be01cbd6a882?action=collection%2Fimport)
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/300b6d41be01cbd6a882?action=collection%2Fimport#?env%5BSandbox%5D=dW5kZWZpbmVk)
```

Comment:
I viewed this file on your branch (https://github.com/plaid/plaid-postman/blob/a4523f13d9553c10c672406b0f58cdf5b9cd85df/README.md) and clicked on this link, which opened a Postman tab in my browser but did not launch the desktop client.

Reply:
thank you for checking! It looks like attempting to export the environment variables is borking postman. It looks like I will have to stop exporting them and then update the directions to reflect that they aren't being exported :-(